spec/controllers/vacancies_controller_spec.rb

```diff
@@ -61,15 +61,22 @@
         end
       end
 
-      context 'search audiotor' do
+      context 'search auditor' do
         let(:params) do
           {
-            subject: 'foo'
+            job_title: 'Should have three match'
           }
         end
 
-        it 'should call the search auditor' do
-          expect(AuditSearchEventJob).to receive(:perform_later)
+        it 'should call the search auditor', elasticsearch: true do
+          3.times { create(:vacancy, job_title: 'Should have three match') }
+          Vacancy.__elasticsearch__.client.indices.flush
```

Comment:
Eeek, we should move this into a nice place, but I'm happy to approve this for now.

Reply:
I was surprised its not part of the teardown, but I found this solution at a few other spec files so I assumed its fine. I would suggest fixing this in another PR since this could involve some refactoring in other spec files e.g: `job_seekers_can_search_vacancies_spec.rb`